app/jane-layers/facilities/filter-components/LayerSelector.jsx

```diff
@@ -63,7 +63,11 @@ function processChecked(layers) {
 class LayerSelector extends React.Component {
   constructor(props) {
     super(props);
-    this.state = { expanded: null };
+    this.state = {
+      expanded: null,
+      initialComponentMount: false,
```

Comment:
Hmmm this feels kinda anti-pattern-y. Why do you need this? Maybe we can accomplish the same effect using one of the component lifecycle hooks?

Reply:
By passing true/false down to the `AreaFilterSelector`,  `{ value: 'nta2020', label: 'Neighborhood Tabulation Area' }` is set as the `selectedLayer` ONLY when`initialComponentMount` is true (which is just the one time). Any option changes (to different filter dimensions) in the `LayerSelector`'s  `updateFilterDimension` function flips that flag to false without affecting the functionality in  either component.

Reply:
I changed the name of the boolean flag to `exploreANeighborhoodSelected` so it is clearer (hopefully) where the flag is coming from. I added a check to see if we have `locationState` when the `LayerSelector` component mounts to avoid setting the flag to `true` if we click on MAP instead of  `EXPLORE A NEIGHBORHOOD` in `SpashDuo`.